BraveWallet/WalletStrings.swift

```diff
@@ -1884,5 +1884,61 @@ extension Strings {
       value: "Go back",
       comment: "The transaction edit error alert button which will dismiss the alert."
     )
+    public static let transactionDetailsTitle = NSLocalizedString(
+      "wallet.transactionDetailsTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Transaction Details",
+      comment: "The title for the transaction details modal."
+    )
+    public static let transactionDetailsTxFeeTitle = NSLocalizedString(
+      "wallet.transactionDetailsTxFeeTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Transaction fee",
+      comment: "The title for displaying the transaction fee of the transaction in the transaction details modal."
+    )
+    public static let transactionDetailsMarketPriceTitle = NSLocalizedString(
+      "wallet.transactionDetailsMarketPriceTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Market price",
+      comment: "The title for displaying the market price of the transaction in the transaction details modal."
+    )
+    public static let transactionDetailsDateTitle = NSLocalizedString(
+      "wallet.transactionDetailsDateTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Date",
+      comment: "The title for displaying the date of the transaction in the transaction details modal."
+    )
+    public static let transactionDetailsNetworkTitle = NSLocalizedString(
+      "wallet.transactionDetailsNetworkTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Network",
+      comment: "The title for displaying the network of the transaction in the transaction details modal."
+    )
+    public static let transactionDetailsTxHashTitle = NSLocalizedString(
+      "wallet.transactionDetailsTransactionHashTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Transaction hash",
+      comment: "The title for displaying the transaction hash in the transaction details modal."
+    )
+    public static let transactionDetailsStatusTitle = NSLocalizedString(
+      "wallet.transactionDetailsStatusTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Status",
+      comment: "The title for displaying the transaction status in the transaction details modal."
```

Comment:
For single words the comment here is really for the translator, not for us. We should be explaining what "Status" refers to and how its used rather than what screen its in and such.

Something like: `The transaction status, which describes the how far along a transaction is to completing. Appears next to the words such as "Approved", "Submitted", "Pending", etc.` is much more useful for translators for languages that have multiple words for a single english word based on context.

This comment is for all the translations added here :)